apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx

```diff
@@ -4,3 +4,27 @@ metaTitle: Placement and order
 mode: draft
 route: /patterns/placement-and-order/
 ---
+## Guidelines
+In order to place yout objects and component the right way, you have always to keep in mind the ready pattern of your user. 2 types of patterns can be encountered: F and Z.
+
+### The F reading pattern
+
+In the F-shaped scanning pattern is characterized by many fixations concentrated at the top and the left side of the page.
+
+Users first read in a horizontal movement, usually across the upper part of the content area. He will then
+move down the page a bit and then read across in a second horizontal movement that typically covers a shorter area than the previous movement. This additional element forms the F’s lower bar.
```

Comment:
A few British spellings, and updates to keep it consistent with the Z reading pattern paragraph: 

>~~In t~~The F-shaped scanning pattern is characteri~~z~~sed by many fixations concentrated at the top and the left side of the page.
> 
> Users first read in a horizontal movement, usually across the upper part of the content area. ~~He~~They will then
> move down the page a bit and then read across in a second horizontal movement that typically covers a shorter area than the previous movement. This additional element forms the F’s lower bar.

Reply:
Updated